### PR TITLE
Adding parameters class for compilers and converters

### PR DESCRIPTION
**Pull request by Paola-Xaq**

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/models/parameter/PythonEnum.java [hunk 1]
```diff
@@ -9,16 +9,16 @@
...
-package org.fundacion.jala.converter.service.pythoncompiler;
+package org.fundacion.jala.converter.models.parameter;
 
-public enum Python {
+public enum PythonEnum {
 
     V2("\"" + System.getProperty("user.dir") + "/thirdparty/python3/python.exe" + "\""),
     V3("\"" + System.getProperty("user.dir") + "/thirdparty/python3/python.exe" + "\"");
 
     private String version;
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/models/parameter/PythonEnum.java:20]: Maybe these space should be deleted

File: src/main/java/org/fundacion/jala/converter/controller/ExtractMetadataController.java [hunk 2]
```diff
@@ -31,7 +34,8 @@
     FileStorageService fileStorageService;
 
     /**
-     * Endpoint for extract metadata
+     * Endpoint for extract metadata.
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/controller/ExtractMetadataController.java:37]: It should be Endpoint to extract metadata or Endpoint for metadata extraction.

File: src/main/java/org/fundacion/jala/converter/controller/ExtractMetadataController.java [hunk 3]
```diff
@@ -31,7 +34,8 @@
     FileStorageService fileStorageService;
 
     /**
-     * Endpoint for extract metadata
+     * Endpoint for extract metadata.
+     * @return String for download file
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/controller/ExtractMetadataController.java:38]: It should be String to download file or String for downloading file.

**Review by jpgonzales27 — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/models/parameter/JavaParameter.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    private JavaVersion javaVersion;
+
+    public JavaParameter(final JavaVersion javaVersion, final String filePath) {
+        super(filePath);
+        this.javaVersion = javaVersion;
+    }
+
+    /**
+     * Gets the version for Java version.
+     * @return javaVersion Python version
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/models/parameter/JavaParameter.java]: Why does java version returns a Python version?

**Review by edsonarios — APPROVED** [review]

**Review by Raymundogs — APPROVED** [review]

**Pull request merged**